### DISCUSSION TITLE
Machine Readable File Size Expectations

### DISCUSSION THREAD

**Discussion by BobAtMedcost** [CMSgov/price-transparency-guide#350] [Q&A] [answered]
What are CMS’s file size expectations for machine-readable files? Our projections are that they’ll be many terabytes per file, plus we’re thinking that we’ll need a file per group/plan. My company is relatively small, but our niche is providing customized service for our clients. I appreciate the recent schema changes to reduce file size such as the Provider Reference object. No doubt that will help some companies, but it assumes that pricing is the same for all providers under a tax id (many have thousands of providers). That’s not the case for us so we won’t be able to use this feature. So far, we have not been able to run complete tests due to limitations on storage and processing power. It will require our company to make major capital outlays to upgrade our IT infrastructure to be able to produce and transfer files of this size. Another foreseen issue is having a processing window large enough to create these files monthly. I’d love to hear what others are projecting for their file sizes, and how they plan to accommodate the public transparency requirements.

**Comment by shaselton-usds** [comment, marked as answer]
File sizes will vary based on many variables that will be unique to your plan's arrangements and whether or not certain strategies will be used when building out the files such as logical groupings of service_codes or taking advantage of provider reference (internal or external) or leveraging a table-of-contents to reuse common in-network pricing across networks and plans.

While other may chime in if they wish to answer your question on their estimated file sizes, it will most likely be relative to their unique set of variable constraints.
